app/views/podcast_planner/_form_date_calculator.html.erb

```diff
@@ -46,25 +36,26 @@
   </div>
 
   <div class="mb-4" data-controller="toggle-field">
-    <h3 class="fw-bold"><%= t(".header.range_select") %></h3>
-    <p><%= t(".help.range_select") %></p>
+    <h3 class="mb-3"><%= t(".header.range_select") %></h3>
     <div class="row mb-4">
-      <div class="col-lg-6">
-        <p><%= t(".help.calculate_range") %></p>
+      <div class="col-12">
         <div>
           <div class="form-floating">
             <%= form.select :date_range_condition, end_condition_options, {}, data: {action: "toggle-field#displayOnlyThisField click#submit"} %>
             <%= form.label t(".label.date_range_condition") %>
           </div>
         </div>
       </div>
+    </div>
+    <div class="row mb-4">
       <div class="col-lg-6">
-        <p><%= t(".help.select_range") %></p>
-          <div class="form-floating mb-2">
-            <%= form.date_field :start_date, min: DateTime.tomorrow, max: DateTime.tomorrow + 1.years, value: DateTime.tomorrow, data: {action: "click#submit"} %>
-            <%= form.label :start_date %>
-          </div>
+        <div class="form-floating mb-2">
+          <%= form.date_field :start_date, min: DateTime.tomorrow, max: DateTime.tomorrow + 1.years, value: DateTime.tomorrow, data: {action: "click#submit"} %>
+          <%= form.label :start_date %>
+        </div>
+      </div>
```

Comment:
I adjusted the layout of this field so that the selector field is full width, with two variable fields appearing below. This is more consistent with our UI patterns.